### PR TITLE
Replaced sys.stderr write with django logging (#1113)

### PR DESCRIPTION
**Pull request by bama4**
Replaced `sys.stderr` with django logging scheme used to address `Poor Logging Practice: Use of a System Output Stream`

**Test Instructions**
Run local tests before merging

### CODE REVIEW

File: siteapp/views.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (t-s) [inline siteapp/views.py:1]: I'm sorry for being pedantic, but may you remove this empty line?

Comment (bama4) [reply siteapp/views.py]: My mistake, was removed.

Comment (t-s) [reply siteapp/views.py]: Got me covered, looks good, thanks

File: siteapp/views.py [hunk 2]
```diff
@@ -1590,8 +1590,12 @@ def send_invitation(request):
     except ValueError as e:
         return JsonResponse({ "status": "error", "message": str(e) })
     except Exception as e:
-        import sys
-        sys.stderr.write(str(e) + "\n")
+        logger.error(
+            event="send invitation",
+            object={"status": "error",
+                    "message": " ".join(["There was a problem -- sorry!", str(e)])},
+            user={"id": request.user.id, "username": request.user.username}
```
Comment (davidpofo) [inline siteapp/views.py:1597]: lgtm

**Review by davidpofo — APPROVED** [review]

**Comment by bama4** [comment]
Test results (Failures do not appear to be related to these changes):

```
======================================================================
FAIL: test_invitations (siteapp.tests.GeneralTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/adangwa/REPOS/govready-q/siteapp/tests.py", line 546, in test_invitations
    do_invitation(self.user2.email)
  File "/home/adangwa/REPOS/govready-q/siteapp/tests.py", line 515, in do_invitation
    self._accept_invitation(username)
  File "/home/adangwa/REPOS/govready-q/siteapp/tests.py", line 373, in _accept_invitation
    self.assertRegex(self.browser.title, "Sign In")
AssertionError: Regex didn't match: 'Sign In' not found in 'Welcome to Compliance Automation - GovReady-Q'

======================================================================
FAIL: test_questions_module (siteapp.tests.QuestionsTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/adangwa/REPOS/govready-q/siteapp/tests.py", line 1233, in test_questions_module
    do_submodule("My second answer.")
  File "/home/adangwa/REPOS/govready-q/siteapp/tests.py", line 1209, in do_submodule
    self.assertRegex(self.browser.title, "Next Question: Introduction")
AssertionError: Regex didn't match: 'Next Question: Introduction' not found in 'Next Question: module | Test The Module Question Types - GovReady-Q'

----------------------------------------------------------------------
Ran 63 tests in 364.902s

FAILED (failures=2)
```

**Comment by gregelin** [comment]
@bama4 I agree. The changes are not related to the changes. I believe both of these tests are solved in other PRs. I know the "FAIL: test_invitations (siteapp.tests.GeneralTests)" is solved in PR #1111.

**Comment by bama4** [comment]
Looks like this is ready to be merged?

**Comment by davidpofo** [comment]
PR party 12/3.

**Review by gregelin — COMMENTED** [review]
Looks good.

**Pull request merged**